.circleci/config.yml

```diff
@@ -80,19 +80,14 @@ jobs:
       - aws-eks/update-kubeconfig-with-authenticator:
           cluster-name: "sit"
           aws-region: "$REGION"
-      - run: |
-          export ENV_NAMESPACE=paymenthub
-          kubectl config use-context arn:aws:eks:$REGION:419830066942:cluster/sit
-          kubectl config get-contexts
-          cd helm/kibana-secret/
-          make secrets || echo "kibana" already exists
-
-      - run: |
-          export ENV_NAMESPACE=paymenthub
-          kubectl config use-context arn:aws:eks:$REGION:419830066942:cluster/sit
-          kubectl config get-contexts
-          cd helm/es-secret/
-          make secrets || echo "elastic-certificates" already exists
+      - run: 
+          name: Bulk secret creation
```

Comment:
we have removed bulk secret creation, please dont add it back. You can remove entire secret creation step.